src/chrome/chromeDebugAdapter.ts

```diff
@@ -114,6 +140,8 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
     private _columnBreakpointsEnabled: boolean;
 
     private _smartStepCount = 0;
+    private _initializedEventFired: boolean = false;
```

Comment:
Instead of keeping a separate flag, you can set the array to null to indicate that events shouldn't be buffered anymore. Have used this pattern in the past.